src/commands/docs/spec/generateDot.spec.ts

```diff
@@ -39,7 +40,14 @@ describe('sasjs doc lineage', () => {
         defaultTarget: 'viya'
       })
 
-      await expect(doc(new Command(`doc lineage`))).resolves.toEqual(0)
+      const { target } = await findTargetInConfiguration(
```

Comment:
would be nice to create a variable to avoid code duplications

Reply:
fixed.